TRB_label_assist/clustering_examples.ipynb

```diff
@@ -26,12 +26,13 @@
                 "%autoreload 2\n",
                 "\n",
                 "from uuid import UUID\n",
-                "\n",
+                "import itertools\n",
```

Comment:
extraneous import

Reply:
done. This was not tested till that point.